test/switchtoactivity_test.js

```diff
@@ -0,0 +1,75 @@
+suite('switchToActivity', function() {
+  var launch = require('../lib/launch').launch;
+  var switchToApp = require('../lib/switchtoapp').switchToApp;
+  var switchToActivity = require('../lib/switchtoactivity').switchToActivity;
+
+  var ACTIVITY_CALLER_DOMAIN = 'activity.caller.gaiamobile.org';
+  var ACTIVITY_CALLEE_DOMAIN = 'activity.callee.gaiamobile.org';
+
+  var apps = {};
+  apps[ACTIVITY_CALLER_DOMAIN] = __dirname + '/apps/activitycaller';
+  apps[ACTIVITY_CALLEE_DOMAIN] = __dirname + '/apps/activitycallee';
+
+  var profile = {
+    settings: {
+      'ftu.manifestURL': null,
+      'lockscreen.enabled': false,
+      'screen.timeout': 0
```

Comment:
I added ```screen.timeout: 0``` because it was the reason of several intermittent failures in gaia (see [1]) + it's going to be the default setting for the test profile as well (see [2]). Probably it makes sense to make add it to ```createClient``` default profile as well.

[1] https://bugzilla.mozilla.org/show_bug.cgi?id=1121766
[2] https://bugzilla.mozilla.org/show_bug.cgi?id=1139631